client/src/app/routes/stardust/Addr.tsx

```diff
@@ -716,7 +716,7 @@ class Addr extends AsyncComponent<RouteComponentProps<AddrRouteProps>, AddrState
             this._tangleCacheService
         );
         const inputsRelated = inputs.filter(input => input.transactionAddress.hex === this.state.address);
-        const outputsRelated = outputs.filter(output => output.address.hex === this.state.address);
+        const outputsRelated = outputs.map(output => output);
```

Comment:
What's the point of this map ? It doesn't change anything.

Reply:
fixed.